### PR TITLE
Rebrand staking page

### LINKED ISSUE #456: Rebrand DAO Dashboard - Staking page
Updates styles of the Staking page so it matches [new designs](https://www.figma.com/design/NG6QClnVrvPRZ5gJ3s0zen/%E2%9F%81--Api3-DAO---Branding-Update?node-id=8676-107801&node-type=section&m=dev). This includes
- Main section of the page (Layout to be done in separate task)
- Unconnected wallet state (we don't have updated video yet, leave the current one)
- Connected wallet state (and some tokens staked)
- Unstaking state

No modals should be updated in this task (Staking / Unstaking). 

Image

### PR DESCRIPTION
**Pull request by Anboias**
Closes #456 

[Designs](https://www.figma.com/design/NG6QClnVrvPRZ5gJ3s0zen/%E2%9F%81--Api3-DAO---Branding-Update?node-id=9003-130567&m=dev)

## What does this do?
* updates main section of the page
* uodates unconnected wallet state
* updates connected wallet state (and some tokens staked)
* updates unstaking state

## What is this not doing?
* doesn't update skeletons as we don't use any yet. Adding them now implies we meddle with the logic (to retrieve the loading/error flags), and I'm not sure if we want that now

#### Pending unstaking

| Mobile | Desktop |
|--------|--------|
| image | image | 

#### Main view
| Unconnected | Connected |
|--------|--------|
| image| image |

### CODE REVIEW

File: src/components/header/header.module.scss [hunk 1]
```diff
@@ -1,51 +1,26 @@
...
 .header {
   position: relative;
-  margin-bottom: 2.5 * $space-xxl;
   pointer-events: none;
+  margin-top: 56px;
 
-  @media (max-width: $max-md) {
-    margin-bottom: $space-xxxl;
+  @media (min-width: $md) {
+    margin-top: 0;
```
Comment (Anboias) [inline src/components/header/header.module.scss:10]: Removing the margin bottom from the page header, leaving it up to each page to decide this spacing. Top margin instead works, as all screens from all pages match with each other on that.

File: src/components/header/header.module.scss [hunk 2]
```diff
@@ -1,51 +1,26 @@
 @import '../../styles/variables.module.scss';
+@import '../../styles/fonts.module.scss';
 
 .header {
   position: relative;
-  margin-bottom: 2.5 * $space-xxl;
```
Comment (Anboias) [inline src/components/header/header.module.scss:5]: Removing the margin bottom from the page header, leaving it up to each page to decide this spacing. 
Top margin on the other hand works, as all screens from all pages match with each other on that.

File: src/pages/dashboard/dashboard.tsx [hunk 3]
```diff
@@ -59,80 +77,83 @@ const Dashboard = () => {
...
+                <p className={styles.cardContentTitle}>unstaked</p>
+                <p className={styles.cardContentValue} data-cy="unstaked">
+                  {data ? formatAndRoundApi3(data.userUnstaked) : UNKNOWN_NUMBER}
+                </p>
+              </div>
+            </div>
+          }
+          contentFooter={
+            !pendingUnstake && (
+              <Button size="sm" type="secondary" onClick={() => setOpenModal('unstake')} disabled={!canInitiateUnstake}>
```
Comment (peterjurco) [inline src/pages/dashboard/dashboard.tsx:151]: Currently we use Figma's "Secondary gradient" as secondary button. These action buttons (Withdraw, Initiate Unstake...) should be "Secondary neutral" variant. Can you add that variant?

Comment (peterjurco) [reply src/pages/dashboard/dashboard.tsx:151]: Also, the button size should be `xs` not `sm`

File: src/pages/dashboard/dashboard.tsx [hunk 4]
```diff
@@ -59,80 +77,83 @@ const Dashboard = () => {
...
+            </div>
+          }
+          contentFooter={
+            <Button type="secondary" size="sm" onClick={() => setOpenModal('withdraw')} disabled={!canWithdraw}>
+              Withdraw
+            </Button>
+          }
+        />
+
+        <Card
```
Comment (peterjurco) [inline src/pages/dashboard/dashboard.tsx:119]: The Card should have dark border when the Unstake was initiated

image

File: src/pages/dashboard/dashboard.tsx [hunk 5]
```diff
@@ -59,80 +77,83 @@ const Dashboard = () => {
...
+          header={
+            <CardHeader>
+              <h5>Staking</h5>
+              <div className={styles.cardHeaderButton}>
+                <Tooltip overlay="You need to deposit API3 tokens before staking">
+                  <img src={images.helpOutline} alt="new proposal help" />
+                </Tooltip>
```
Comment (peterjurco) [inline src/pages/dashboard/dashboard.tsx:126]: Could you also restyle all these tooltips on the page?

image

File: src/pages/dashboard/pending-unstake-panel/pending-unstake-panel.tsx [hunk 6]
```diff
@@ -77,55 +76,47 @@ const PendingUnstakePanel = (props: Props) => {
...
+          <p className={styles.pendingUnstakeName}>Amount</p>
+          <h5 data-cy="amount">{formatAndRoundApi3(amount)}</h5>
+        </div>
+        <div className={classNames(styles.pendingUnstakeRow, { [globalStyles.tertiaryColor]: canUnstake })}>
+          <p className={styles.pendingUnstakeName}>Cooldown</p>
+          <div className={styles.pendingUnstakeCountdown}>
+            <div className={styles.pendingUnstakeCountdownItem}>
+              <p className={classNames(globalStyles.textSmall, globalStyles.medium)}>{timerDays}</p>
+              <p className={styles.shortcut}>D</p>
+            </div>
```
Comment (peterjurco) [inline src/pages/dashboard/pending-unstake-panel/pending-unstake-panel.tsx:92]: You should remove all global styles from here, they are either unnecessary or incorrect
```suggestion
        <div className={styles.pendingUnstakeRow}>
          <p className={styles.pendingUnstakeName}>Cooldown</p>
          <div className={styles.pendingUnstakeCountdown}>
            <div className={styles.pendingUnstakeCountdownItem}>
              <p>{timerDays}</p>
              <p className={styles.shortcut}>D</p>
            </div>
```

**Review by peterjurco — CHANGES_REQUESTED** [review]
Looks promising, but couple of things should be addressed

File: src/pages/dashboard/dashboard.tsx (hunk 3 above)
Comment (Anboias) [reply src/pages/dashboard/dashboard.tsx:151]: Done. I also created the `/variants` folder to hold the styles separately. I would like to move your recent file `link-styles.module.scss` in there too, you're okay with that?

image

**Review by mcoetzee — APPROVED** [review]
Nothing to address from my side 👍

File: src/pages/dashboard/pending-unstake-panel/pending-unstake-panel.tsx (hunk 6 above)
Comment (Anboias) [reply src/pages/dashboard/pending-unstake-panel/pending-unstake-panel.tsx]: Did that and also fine-tuned the counter box a bit, it was not aligned properly with the amount

image

File: src/pages/dashboard/dashboard.tsx (hunk 5 above)
Comment (Anboias) [reply src/pages/dashboard/dashboard.tsx:126]: This is still work in progress, I'll ping you when it's done

File: src/pages/dashboard/dashboard.tsx (hunk 3 above)
Comment (peterjurco) [reply src/pages/dashboard/dashboard.tsx]: Good job, it's definitely better this way. Go ahead and move the link styles there as well 👍

File: src/components/button/variants/secondary-neutral.module.scss [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+    @include font-button-1;
+  }
+
+  &:hover,
+  &:active,
+  &:disabled {
+    &:before {
+      padding: 0px;
+    }
+  }
```
Comment (peterjurco) [inline src/components/button/variants/secondary-neutral.module.scss:48]: You can remove this - it's in the secondary button to remove the gradient border but here we don't have any so this is doing nothing

**Review by peterjurco — APPROVED** [review]

File: cypress/e2e/delegation.cy.ts [hunk 8]
```diff
@@ -5,7 +5,7 @@ describe('delegation', () => {
     cy.resetBlockchain().login();
 
     // Approve, deposit and stake
-    cy.findByText('+ Deposit').click();
+    cy.dataCy('staking-cards').findByRole('button', { name: 'Deposit' }).click();
```
Comment (mcoetzee) [inline cypress/e2e/delegation.cy.ts:8]: Lets rather use the built in `data-testid="..."` and `cy.findByTestId(...)` for all of these `cy.dataCy()` usages that you've added.

Comment (Anboias) [reply cypress/e2e/delegation.cy.ts]: Done

**Pull request merged**